src/pybind/mgr/dashboard/services/cluster.py

```diff
@@ -44,6 +48,35 @@ def from_db(cls):
     @classmethod
     def get_capacity(cls) -> ClusterCapacity:
         df = mgr.get('df')
+        total_objects = 0
+        total_pool_bytes_used = 0
+        average_object_size = 0
+        rgw_pools = cls.get_rgw_pools()
+        for pool in df['pools']:
+            if pool['name'] in rgw_pools:
+                total_objects = total_objects + pool['stats']['objects']
+                total_pool_bytes_used = total_pool_bytes_used + pool['stats']['bytes_used']
+        if total_objects and total_objects != 0:
```

Comment:
This should be enough I think.
```suggestion
        if total_objects != 0:
```

Reply:
Done